src/swap/components/SwapSettings.tsx

```diff
@@ -0,0 +1,37 @@
+import { isValidElement, useMemo } from 'react';
+import { SwapSettingsSvg } from '../../internal/svg/swapSettings';
+import { cn } from '../../styles/theme';
+import type { SwapSettingsReact } from '../types';
+
+export function SwapSettings({
+  className,
+  icon,
+  title = 'Auto',
```

Comment:
What Auto means in this context. In my mind a setting button does not need text at all.

Reply:
Agreed. Removing text as the default.